frontend/src/api.ts

```diff
@@ -343,19 +344,71 @@ export function getCalendarRecipeList({
   start,
   end
 }: {
-  teamID: TeamID
-  start: string
-  end: string
+  readonly teamID: TeamID
+  readonly start: string
+  readonly end: string
 }) {
   const id = teamID === "personal" ? "me" : teamID
-  return http.get<ICalRecipe[]>(`/api/v1/t/${id}/calendar/`, {
+  return http.request({
```

Comment:
I messed around with overloads so we could do `http.get(url, { shape, params})` but it was rather verbose for not much gain